mitodl/mit-open
# Updated designs for the unit page

### What are the relevant tickets?

Closes: https://github.com/mitodl/hq/issues/4976

### Description (What does it do?)

Applies [current designs](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=3769-55562&m=dev) to the units page.

Unit cards are updated to the desktop card design and display the heading text in place of sub-heading text.

At mobile screen widths the designs are preserved and continue to display the subheading, pending any updates to the [designs for mobile](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=4083-93794&m=dev).

### Screenshots (if appropriate):


<img width="1286" alt="image" src="https://github.com/user-attachments/assets/d46c0409-c62e-4eb3-b048-0ac16250f6e7">

---

<img width="498" alt="image" src="https://github.com/user-attachments/assets/f3e52058-15b8-404e-92db-082e00531852">


### How can this be tested?

Navigate to `/units/` and confirm that the desktop designs and text updates are applied correctly.
